localstack/services/dynamodbstreams/dynamodbstreams_api.py

```diff
@@ -93,7 +93,7 @@ def post_request():
             ShardId=stream_shard_id, ShardIteratorType=data['ShardIteratorType'])
     elif action == '%s.GetRecords' % ACTION_HEADER_PREFIX:
         kinesis_records = kinesis.get_records(**data)
-        result = {'Records': []}
+        result = {'Records': [], 'NextShardIterator': kinesis_records['NextShardIterator']}
```

Comment:
Can we assume that this value is always present (i.e., does not raise `KeyError`)? I'd prefer to lean on the safe side and replace this with:
```
result = {'Records': [], 'NextShardIterator': kinesis_records.get('NextShardIterator')}
```